frontend/src/components/@common/MessageTextarea/MessageTextarea.tsx

```diff
@@ -35,7 +35,7 @@ const MessageTextarea = ({
         <Label className={styles.label} required={required}>
           {label}
         </Label>
-        {description && <Description>{description}</Description>}
+        {description && <Description className={styles.description}>{description}</Description>}
```

Comment:
```suggestion
        {description && (
          <div className={styles.description} dangerouslySetInnerHTML={{ __html: description }} />
        )}
```

r: 구글 번역 위젯을 적용하면 뉴라인이 사라지네요... 🥲

과제 미리보기와 비슷하게 html 을 삽입하는 것으로 변경하면 좋을 것 같습니다.
다행히 `MesasgeTextArea` 를 사용하는 컴포넌트가 많지 않고, 이렇게 변경했을 때 로컬에서는 잘 동작하는 것을 확인했는데요.

어떤 방식으로 진행할지 오후에 결정하면 좋겠습니다!